jupyterbook/_toc.yml

```diff
@@ -0,0 +1,81 @@
+          - file: content/Code Gallery/data_access/2017-01-18-siphon-explore-thredds.ipynb
+          - file: content/Code Gallery/data_access/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb
+          - file: content/Code Gallery/data_access/2017-06-12-NCEI_RA_archive_history.ipynb
+          - file: content/Code Gallery/data_access/2017-07-25-HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2017-12-15-finding_HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2018-03-01-erddapy.ipynb
+
+    - file: content/Code Gallery/data_analysis_and_visualization/data_analysis_and_visualization.md
+      sections:
+          - file: content/Code Gallery/data_analysis_and_visualization/2016-12-22-boston_light_swim.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2020-10-10-GTS.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2020-02-14-QARTOD_ioos_qc_Water-Level-Example.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2017-09-09-hurricane_irma.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2019-02-26-hurricane_gis_part01.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2019-05-27-hurricane_gis_part02.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2018-12-04-grids.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2019-03-08-grids-temperature.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2018-03-15-ssh-skillscore.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2018-03-30-wave_height_assessment.ipynb
+          - file: content/Code Gallery/data_analysis_and_visualization/2016-11-15-glider_data_example.ipynb
+
+    - file: content/Code Gallery/data_publication/data_publication.md
+      sections:
+        - file: content/Code Gallery/data_publication/2016-12-19-exploring_csw.ipynb
+        - file: content/Code Gallery/data_publication/2016-11-16-CF-UGRID-SGRID-conventions.ipynb
+        - file: content/Code Gallery/data_publication/2020-12-08-DataToDwC.ipynb
+        - file: content/Code Gallery/data_publication/2018-02-27-pocean-timeSeries-demo.ipynb
+        - file: content/Code Gallery/data_publication/2017-11-01-Creating-Archives-Using-Bagit.ipynb
+    
+    - file: content/Code Gallery/R_Matlab_Notebooks/R_Matlab_Notebooks.md
+      sections:
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-01-23-R-notebook.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-03-30-octave_notebook_example.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-08-01-xtractoR.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2018-02-20-obis.ipynb
+        - file: content/Code Gallery/R_Matlab_Notebooks/2017-11-30-rerddap.ipynb
+
+
+- part: Video Tutorials
```

Comment:
@MathewBiddle I wonder if we should still have the videos in the JB. They are helpful but seems a bit odd and kind of mixed[content-out-if-context.

Reply:
I don't know just yet. I really like [this resource](https://www.esipfed.org/data-help-desk-at-egu-2021) that ESIP put together for the data help desk at EGU-2021. If possible, I would like to replicate it with our videos. I'm just not sure how to just yet...